src/celeritas/phys/ParticleTrackView.hh

```diff
@@ -340,7 +340,7 @@ CELER_FUNCTION units::MevMomentumSq ParticleTrackView::momentum_sq() const
 {
     const real_type energy = this->energy().value();
     real_type result = energy * energy + 2 * this->mass().value() * energy;
-    CELER_ENSURE(result > 0);
+    CELER_ENSURE(result >= 0);
```

Comment:
With integral approach off this failed when a positron sampled ionization before coming to rest and momentum was calculated in the Moller-Bhabha interactor constructor.

Reply:
That should still be positive for a positron right? The only thing with zero momentum should be a photon "at rest"...

Reply:
Hmm if the kinetic energy is zero then the momentum should be zero, right?

Reply:
Oh goodness, I was thinking total energy. You're quite right, ignore me!